src/main/java/com/naveenAutomation_Browsers/Browser.java

```diff
@@ -5,7 +5,6 @@ public enum Browser {
 	CHROME("Chrome"), 
```

Comment:
thanks

Reply:
Opera browser is hard to use and it crashes all the time. 
Thank you!